src/main/java/com/google/api/codegen/transformer/InitCodeImportSectionTransformer.java

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2017 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.transformer;
+
+import com.google.api.codegen.metacode.InitCodeNode;
+import com.google.api.codegen.viewmodel.ImportSectionView;
+
+/** Generates an ImportSectionView for the InitCodeTransformer. */
+public interface InitCodeImportSectionTransformer {
```

Comment:
What exactly changes are needed if you combine these two interfaces? From the architecture perspective it is expensive to have one extra top-level interface which is only used by Python (And it only has one method). We should avoid adding interfaces with similar functionality if not necessary because it is hard to know the difference without looking into the implementation details.